Packs/CommonScripts/Scripts/CloseInvestigationAsDuplicate/CloseInvestigationAsDuplicate.py

```diff
@@ -1,7 +1,27 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
-current_incident_id = demisto.incidents()[0]['id']
-duplicate_id = demisto.args()['duplicateId']
-res = demisto.executeCommand("linkIncidents", {"incidentId": duplicate_id, "linkedIncidentIDs": current_incident_id,
-                                               "action": "duplicate"})
-demisto.results(res)
+
+
+NO_INCIDENT_CLOSED_MSG = '0 incidents marked as duplicates'
+
+
+def is_incident_not_closed(response):
+    """Check if the incident was closed based on the response contents."""
+    content = response[0].get('Contents') or response[0].get('contents')
+    return content == NO_INCIDENT_CLOSED_MSG
+
+
+def main():
+    current_incident_id = demisto.incidents()[0]['id']
+    duplicate_id = demisto.args()['duplicateId']
+    res = demisto.executeCommand("linkIncidents", {"incidentId": duplicate_id, "linkedIncidentIDs": current_incident_id,
+                                                   "action": "duplicate"})
+    raise_error = demisto.args().get('raise_error', False)
+    if raise_error and is_incident_not_closed(res):
```

Comment:
The failure message when the server returns the message **"0 incidents marked as duplicates"** should be present even when **raise_error** is **false**, just without raising an error.

Reply:
Done